### PR TITLE
Upgrade flow-bin to 0.71.0

### PR DESCRIPTION
**Pull request by bhongy**

### CODE REVIEW

**Comment by christianvuerings** [comment]
Deploy preview for *gestalt* ready!

Built with commit a56675c879e1b22c357cde21ae7446808a5da1d9

https://deploy-preview-155--gestalt.netlify.com

**Comment by pinterest-ios** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Please add a [changelog.md](https://github.com/pinterest/gestalt/blob/master/CHANGELOG.md) entry for your changes.
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: packages/gestalt/src/Column/Column.js [hunk 1]
```diff
@@ -31,10 +31,10 @@ export default function Column(props: ColumnProps) {
     props.sm && styles[`smCol${props.sm}`],
     props.md && styles[`mdCol${props.md}`],
     props.lg && styles[`lgCol${props.lg}`],
-    props.span && styles[`xsCol${props.span}`],
-    props.smSpan && styles[`smCol${props.smSpan}`],
-    props.mdSpan && styles[`mdCol${props.mdSpan}`],
-    props.lgSpan && styles[`lgCol${props.lgSpan}`]
+    props.span != null && styles[`xsCol${props.span}`],
```
Comment (bhongy) [inline packages/gestalt/src/Column/Column.js]: `Columns` can be literal number `0` which is falsy - previously we wouldn't hit `.xsCol0` classname.

File: packages/gestalt/src/Column/Column.js [hunk 2]
```diff
@@ -31,10 +31,10 @@ export default function Column(props: ColumnProps) {
     props.sm && styles[`smCol${props.sm}`],
     props.md && styles[`mdCol${props.md}`],
     props.lg && styles[`lgCol${props.lg}`],
-    props.span && styles[`xsCol${props.span}`],
```
Comment (bhongy) [inline packages/gestalt/src/Column/Column.js]: `Columns` can be literal number `0` which is falsy - previously we wouldn't hit `.xsCol0` classname.

File: docs/src/Touchable.doc.js [hunk 3]
```diff
@@ -48,7 +48,8 @@ card(
       },
       {
         name: 'onTouch',
-        type: '({ event: SyntheticMouseEvent<> }) => void',
+        type:
+          '({ event: SyntheticEvent<> | SyntheticKeyboardEvent<> }) => void',
         required: true,
```
Comment (bhongy) [inline docs/src/Touchable.doc.js]: Doc states that this prop is required but it was (flow) annotated as optional. Some internal call sites currently treat this as optional.

Comment (bhongy) [reply docs/src/Touchable.doc.js]: I'll leave the doc update to another PR since it's not directly relevant to this change.

**Pull request merged**